### PR TITLE
lang: adding arabic translation

### PR DESCRIPTION
**Pull request by BoFFire**
Adding arabic translation made with Weblate.
https://trans.openalgeria.org/projects/jitsi/jitsi-meet/

### CODE REVIEW

**Comment by BoFFire** [comment]
Document signed !

File: lang/languages.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "en": "English",
     "af": "Afrikaans",
+    "ar": "Arabic"
```
Comment (luixxiul) [inline lang/languages.json:4]: needs `,`

Comment (BoFFire) [reply lang/languages.json]: Fixed. Thank you !

**Comment by luixxiul** [comment]
Also please check https://github.com/jitsi/jitsi-meet/pull/6679/files#diff-c2a31f269ba5f30dfabe9049972beafaR15. There is another lint error.

**Comment by luixxiul** [comment]
don't worry... I will try to take care of your commits 😄

**Comment by BoFFire** [comment]
Removing path to arabic and fallback to origin.

**Comment by BoFFire** [comment]
Closing. I cannot find out where the problem is happening with a trailing space.

**Pull request closed without merging**